### PR TITLE
Database signup

### PR DESCRIPTION
**Pull request by sarinalalria**

### CODE REVIEW

**Comment by sarinalalria** [comment]
Had to change dashbaord and signup - will fix when ready to merge.

**Comment by sarinalalria** [comment]
professor said to add an .env_example of what I did. When testing yourself and adding your own .env, follow the .env_example file.

**Review by nscheiber — APPROVED** [review]

**Review by Sinclair198 — APPROVED** [review]

**Review by YiShianLiu — APPROVED** [review]

File: express-mongodb/signup-express.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  
+    var data = {
+        "firstname": firstname,
+        "lastname": lastname,
+        "phonenumber":phonenumber,
+        "emailID":emailID,
+        "passwordID":passwordID
+       
+    }
+db.collection('users').insertOne(data,function(err, collection){
```
Comment (zeshengchen) [inline express-mongodb/signup-express.js]: Indentation!!!

Comment (sarinalalria) [reply express-mongodb/signup-express.js]: fixed it

**Pull request merged**